region/lowernorfair/east/Pillar Room.json

```diff
@@ -240,6 +240,17 @@
         ]}
       ],
       "reusableRoomwideNotable": "Lower Norfair Pillar Room with Two Power Bombs",
+      "unlocksDoors": [
```

Comment:
Is the issue that another strat could be used to unlock the doors, like [1,2]->[2,2]?
I think an obstacle should be added to get around that instead.
Or is it a bit of both and there is an implicit self linking strat that can open the door.

Reply:
I also don't understand the default ammo and heatFrame costs of unlocksDoors.  And if I should actually be using `"useImplicitRequires": false` if I want a lower heatFrame count than default.

Reply:
Right, by default there is an implicit self-linking strat at each door that can unlock a red door with 50 heat frames, or a yellow door with 110. Since that would be unsound here (due to not accounting for the extra acid damage), we want to remove those implicit strats, by setting `"useImplicitDoorUnlocks": false` on the two door nodes. I think that should be enough to solve the problem, so no need to add an obstacle, or at least I'm not seeing yet what an obstacle would be for?

The implicit requirements in `unlocksDoors` only covers ammo requirements, along with the `Morph` item requirement in the case of using a Power Bomb. So you don't need to use `"useImplicitRequires": false` when putting in heat frames. Also, heat frames shouldn't ever be omitted here unless there aren't any heat frames, because there are no implicit/default heat frames in `unlocksDoors` (only in the implicit self-linking unlock strat). Example situations where you would want to use `"useImplicitRequires": false` would be if using artificial morph to open a Power Bomb door, or where a Power Bomb from another part of the strat (e.g. a Crystal Flash) already opens the door as a side effect.

Reply:
It was the `leaveWithRunway` strats I was worried about, but they have `comeInNormally` to take care of that.